decidim-core/app/helpers/decidim/decidim_form_helper.rb

```diff
@@ -12,10 +12,10 @@ module DecidimFormHelper
     # Returns a String.
     def decidim_form_for(record, options = {}, &)
       options[:data] ||= {}
-      options[:data].update(abide: true, "live-validate" => true, "validate-on-blur" => true)
+      # options[:data].update(abide: false, "live-validate" => false, "validate-on-blur" => false)
```

Comment:
Even when abide and live validate are disabled, I still see the error when I open the conversations view:
![Error visible on page load](https://user-images.githubusercontent.com/864340/185085095-f5bae384-99bb-4617-8e8a-1a9eb6e5e362.png)

Is this the same problem that was discussed regarding the sign up form errors?

The intended behavior is that the error message should not show up directly when this view is loaded. It should only show up after there is actually an error on the field or when blurring out of the field once it was already active before.

Reply:
That looks better already but when you focus in the text element, the error should not pop up instantly. It should only pop up there when you left the field empty and then blurred out from the field. This way the user has done already an action on that field which caused it to change to invalid.

If the error shows up when you focus the field, it's very confusing for the screen reader users as they instantly start looking where the error might be. The error state should only show up when there is actually an error in the user input or the user did something that caused the error to show up.

Reply:
Hmm, that would implicate javascript also. I was trying to find a simpler solution, but...
Our original idea related to this is to bring back [the isolated abide script](https://www.npmjs.com/package/vanilla-abide). Since that's not straightforward, I'll track it down in the pending conversations issue (even though actually it's related to every form) and let this as it is meanwhile.